etc/notebooks/test/test_plotting_libs.ipynb

```diff
@@ -385,7 +386,7 @@
    },
    "outputs": [],
    "source": [
-    "show(p)"
+    "handle = show(p, notebook_handle=True)"
```

Comment:
I do not see the error mentioned above with the new notebook. The intiial bokeh plot appears. However, the ipywidgets associated with it have no effect on its output.